QUICKSTART.md

```diff
@@ -168,37 +159,32 @@ The Swiss Army Kube repository has three main directories that provide a minimal
 * `docs`    - more detailed documentation and various FAQ's
-* `example` - directory to be used as a template for your projects that includes configuration files for modules and variables.
+* `examples` - directories with different types of SAK use-cases to be used as a template for your projects that includes configuration files for modules and variables.
 * `modules` - Terraform modules (must-have and optional) to deploy your cluster with.
 
 ### Project Structure (Example Directory) 
 
-The `swiss-army-kube/example` directory is a boilerplate for your projects. Use it as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.  
+The `swiss-army-kube/examples` directory are a boilerplates for your projects, use them as a template. You can rename it to your project name for convenience. Make as many projects as you need by cloning and modifying this directory.
 
-To configure your project cluster for deployment, just [include modules](./modules) that you need and [set variables](./example/CONFIGURE.md) in the [`.tfvars` file](./example/example.tfvars) before deploying your EKS cluster with Terraform commands. 
+To configure your project cluster for deployment, just [include modules](./modules) that you need and [set variables](./examples/CONFIGURE.md) in the `.tf` files before deploying your EKS cluster with Terraform commands. 
 
-The example directory contains a set of `.tf` files: 
+The `examples/common` directory contains a set of `.tf` files: 
 
-* `main.tf`        - data from modules
-* `local.tf`       - contains local data to use in main.tf
-* `modules.tf`     - a list of modules and their redefined values
-* `output.tf`      - contains ouput definitions for your modules
+* `main.tf`        - main file with infrastructure code
 * `providers.tf`   - list of providers and their values
-* `variables.tf`   - definition of variables used in modules and their default values
-* `example.tfvars` - (rename into `terraform.tfvars` for convenience) - list of values for variables that you modify. 
 
 <a name="adddevs"></a>
 ## Adding Developers to Kubernetes Cluster
  
 ### DevOps engineer steps
 
 1. Add an IAM user in AWS Console for developer with programmatic access
-2. Add user ARN and name to `user_arns` variable with group `system:developers` in `terraform.tfvars` file
+2. Add user ARN and name to `user_arns` variable with group `system:developers` for _kubernetes_  module in `main.tf` file
 3. Run `terraform plan -out=plan` and review
 4. Run `terraform apply plan`
 5. Send `kubeconfig_internal-projects` config and IAM user tokens to the developer
 
-NOTE: To change developers permissions on Kubernetes cluster manipulate `cluster_roles` variable in the `terraform.tfvars` file.
+NOTE: To change developers permissions on Kubernetes cluster manipulate `cluster_roles` variable in the `main.tf` file.
```

Comment:
NOTE: To change developers' permissions on the Kubernetes cluster edit the `cluster_roles` variable in the `main.tf` file.